fiberi18n/i18n.go

```diff
@@ -43,6 +41,8 @@ func (c *Config) loadMessage(filepath string) {
 }
 
 func (c *Config) loadMessages() *Config {
+	c.mu.Lock()
+	defer c.mu.Unlock()
```

Comment:
if you use the sync map there is no reason to use locking

Reply:
but if not use, test will fail because `data race`